.github/workflows/ci.yml

```diff
@@ -34,4 +34,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - run: cd neurow && docker build .
+      - run: |
+          cd neurow 
+          docker build . -t neurow:${{ github.sha }}
```

Comment:
Is there a way not to build the image twice ? (One time for the check + one time for the actual push step)

Reply:
This workflow runs on every PR, and it's just about building the image (`ci.yml`). The one that pushes runs only on main (`docker_push.yml`).

Reply:
So this step ensure that container image from a PR can be build and started properly.